app/assets/scss/application.scss

```diff
@@ -1,49 +1,31 @@
-// GOVUK Front-end toolkit styles, including dependencies for govuk-elements
-// Settings (variables)
-@import "colours";                                // Colour variables
-@import "font_stack";                             // Font family variables
-@import "measurements";                           // Widths and gutter variables
-
-// Mixins
-@import "conditionals";                           // Media query mixin
-@import "device-pixels";                          // Retina image mixin
-@import "grid_layout";                            // Basic grid layout mixin
-@import "typography";                             // Core bold and heading mixins
-@import "shims";                                  // Inline block mixin, clearfix placeholder
-
-// Mixins to generate components (chunks of UI)
-@import "design-patterns/alpha-beta";             // Only required if using _phase-banner.scss
-@import "design-patterns/buttons";                // Only required if using _buttons.scss
-
-// Functions
-@import "url-helpers";
+// GOVUK Front-end toolkit variables, mixins and functions
+@import "colours";
+@import "_grid_layout.scss";
+@import "_typography.scss";
+@import "_url-helpers.scss";
```

Comment:
I think I'd rather keep the inline comments next to these. Even if it's a bit redundant, the extra few words give you a bit more context (eg which are mixins, which are vars, etc).

Reply:
No problem, I added them back in